### PR TITLE
FLA-389: Prevent submit if no credit card registered with CSO

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- JIRA: https://justice.gov.bc.ca/jira/browse/FLA-389
- Prevent submit if no credit card registered with cso

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- local
- jest

### CODE REVIEW

File: src/frontend/efiling-frontend/src/components/page/payment/Payment.js [hunk 1]
```diff
@@ -57,23 +57,35 @@ const submitPackage = (submissionId) => {
     .catch((err) => errorRedirect(sessionStorage.getItem("errorUrl"), err));
 };
 
+const checkSubmitEnabled = (paymentAgreed, setSubmitBtnEnabled) => {
+  if (paymentAgreed && sessionStorage.getItem("cardRegistered") === "true") {
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/payment/Payment.js:61]: TERNARY OPERATOR CHALLENGE?

File: src/frontend/efiling-frontend/src/components/page/payment/Payment.js [hunk 2]
```diff
@@ -57,23 +57,35 @@ const submitPackage = (submissionId) => {
...
+    setSubmitBtnEnabled(true);
+  } else setSubmitBtnEnabled(false);
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/payment/Payment.js:63]:
```suggestion  
  const isEnabled = paymentAgreed && sessionStorage.getItem("cardRegistered") === "true"
      ? true
      : false;
      
  setSubmitBtnEnabled(isEnabled);
```

something like this you mean? @akroon3r

Comment (akroon3r) [reply src/frontend/efiling-frontend/src/components/page/payment/Payment.js]: YAR!

File: src/frontend/efiling-frontend/src/components/page/payment/Payment.js (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-frontend/src/components/page/payment/Payment.js]: complete

**Comment by akroon3r** [comment]
:shipit: ⚓

File: src/frontend/efiling-frontend/src/components/page/payment/Payment.js [hunk 3]
```diff
@@ -59,9 +59,7 @@ const submitPackage = (submissionId) => {
 
 const checkSubmitEnabled = (paymentAgreed, setSubmitBtnEnabled) => {
   const isEnabled =
-    paymentAgreed && sessionStorage.getItem("cardRegistered") === "true"
-      ? true
-      : false;
+    paymentAgreed && sessionStorage.getItem("cardRegistered") === "true";
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/payment/Payment.js:62]: Consider yourself....linted

**Pull request merged**